extension-js/extension.js
# Fix content_scripts not running in the main world

Fix #66

Depends on #102

## How to test

Use any existing `content_script` and add set it to the "MAIN" world, like:

```json
  "content_scripts": [
    {
      "world": "MAIN",
      "matches": ["https://extension.js.org/*"],
      "js": ["./content/content.ts"]
    }
  ]
```

And ensure it can be reloaded as usual.
